ui/src/app/components/runs/run-detail/run-detail.component.ts

```diff
@@ -44,6 +45,15 @@ export class RunDetailComponent implements OnInit, OnDestroy {
       this.loading = state.detail.loading;
       this.jobInstances = state.detail.jobInstances;
     });
+    this.onRefresh();
+  }
+
+  onRefresh() {
+    this.refreshSubject.subscribe((response) => {
```

Comment:
I think you should also unsubscribe in `ngOnDestroy`

Reply:
Unfortunately unsubscribing on ngOnDestroy destructs the main runs component view. I will leave it as is for now.